sirepo/srunit.py

```diff
@@ -621,3 +627,27 @@ def __req(self, route_or_uri, params, query, op, raw_response, **kwargs):
                     pkdexc(),
                 )
             raise
+
+
+class _UwsgiClient(PKDict):
+
+    SR_SIM_TYPE_DEFAULT = MYAPP
```

Comment:
This is duplicated. Deprecate SR_SIM_TYPE_DEFAULT in TestClient, and move it as a global, since it's only used in srunit and conftest.

Reply:
https://github.com/radiasoft/sirepo/commit/66da4701448010669a6cbfea717df20863acec62